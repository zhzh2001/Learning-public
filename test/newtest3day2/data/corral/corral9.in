123 199
223387723285390559 383797574980585647
24503436769354096 608791513490273629
683067406128554216 779199122595836911
525717583592116593 233497200692638579
473086688094658563 277847440701996529
586619678356984241 404202982499473219
177976333376609526 426918463343077064
87911408548005051 724723834522890169
982378438307393813 266075414509581832
597213083175150985 68295401094368017
119547232546725988 49625453838629682
315811905383601576 770217983007412542
91942005044068598 159998161391862247
561763902331165290 543863624434365901
993472518119475007 283336382730468689
117902915389320281 413271083667144350
99984193244715381 724845960013575117
831177141543720352 866988147932094227
811086703705616384 796492692549823063
392902208129723702 82595713159904737
423598631875796045 543551620694856801
291214181309722959 560549528581910319
764982073810362706 345835728316769657
259559378291739012 715547909244217057
395185390393669207 380383421846097239
663694141986932992 907036237049895058
645284108718956441 826143002281500407
563456475637548346 539648372839430374
359438515780660840 153242315588513338
657500497767361 351766829738729334
603335439506622202 261855959910996070
333031943181126739 194260523179135411
348075602679261629 491079799318643180
115158929749952775 349928246628698730
878488557446390183 412222961574861989
25296324661855509 756669712170408012
680829009367749025 754472400465689075
780814620078913123 494188376710316732
836370816677440301 131471071565360964
518346295244790079 983212754884065196
631634832644374596 256364814366198931
832680107208064527 257804883169490431
370941670025672923 16662902529524764
676226020508050347 791667462100269696
914092151953067915 735505809654607683
916281729873882122 857575806397100155
449552237972392456 852737724249897927
386478921479914994 195304893280170462
184391628521257633 943939401494850735
810227304412222248 758335329985814942
26015607376645759 473424652950624715
977665512022597749 802671337341290099
854023630668723070 140712423668371233
505345555265304575 679354517601442919
892656003489915225 617571162341408569
941555591984480666 446601772542836544
334096799970875427 236245193896784265
170031544464165576 636783254081668445
719913704388570245 13644531856685275
170036514188311879 729761141608045510
621565181754745591 812991564120114037
974580769547659904 420869164531364693
137315035298582018 347048240030541073
652519913026433950 613955765452744308
539235357419362040 586269040060730901
931249400273039007 45119529229757040
623042828458071525 75987758524874044
396990454712588534 6085139241272235
638014197157040179 909786307057535196
429364680044253639 649710816271305139
192896114879847788 980347758398525238
500339267075160624 450316134376425975
189817508317231011 386584947985445695
828344910014421544 856990278842439532
518716735120687102 844532797991650168
205382172717544427 943291468896915523
779253660699249509 746399601428609603
365857489895209541 490208162563193233
829094644637856089 816527795496254519
786566789960341457 694027679236264264
822481724574466990 841884749072680636
61134898628928747 984615950225337972
38878831613806052 570402476739874764
395241507039526626 693798296621282325
343210459805164860 652247718623226567
93162967865579480 634377690753507123
915640382052801090 639778492128207750
115607899426254580 449362217976904124
604876953821172396 257137872758921275
572690753293265183 938647289744777738
530613773281311867 564089462552878134
122808161831285965 555622822553817121
54569317656311933 607800193152532474
388828518499686539 521963809742758391
754898615506891718 960616390635951027
35237398439806105 181051441373784862
240536934998989720 409837781846689909
713316601964249325 5473742920253949
570560861457033655 318014809638477162
224003131466588755 503778412724093937
601648972045568703 749278493382882286
670066524402501074 583931936076497766
928350854929154907 434269595744195986
158838757428849520 847615339098132847
302103785114004837 515774173141787215
542463667789843319 826332745226883653
727090623695174989 736526026983040613
600981796263891337 731979023760580928
881536125784663646 428166169035471260
241074537147275582 50704427307379847
839781789102836391 95106845123279313
45096254856696134 202497711837607396
885233920439571690 675218998505639620
551527185815083005 348068646354333159
859540709410276313 208827228772790429
319055412714576064 633746872991385010
203654972419098770 754037833109683462
872204341456666122 985949573726088150
483766987913852085 618619838534287270
732033918378378977 579458954681034864
470021257529021690 423868831087466071
227918696248399628 419188843721015913
84672980472690096 887707732639245118
854117007358314071 646593832872813826
821923822585729169 155505979766191791
293872704448152972 358090118411395367
323607707812305386 53074347792433548
685365626425252805 38239494018482196
302702959006559609 561550165541306214
897244716742196510 43957689801139410
283650379513934958 600211321526082100
802907098556208408 928599691200150268
468838001467859417 709078316536046653
938122184927818184 193637628196808164
21219821575253775 440798174394190277
212638843852178886 484535785316038651
765091225185776911 481073721808429883
573113368828220075 954075303916632946
258470847661395535 384470897413112543
660357103333884297 463627851505798826
994071868121618181 399104185281185458
739840590199013317 319910396384854675
808295411190772356 394045105049293107
293763082731727980 51878875746224119
603769542964454135 495368969652188992
749083522992145137 791785470494518421
698013140157687423 302485766604938505
801048222048852577 377819527492647005
265892159695696458 7853645806459065
892416318469418122 822845884643605493
81174610489671054 229481756056049994
54035579178801237 377480733633559615
580602839531706986 819051092316187939
50194540927922156 599910887321358864
620122121237672659 155391343224360895
11919111982345329 139501269286486261
460753201213212705 703058690026277316
771071607198176250 776936648231115149
79927454602568009 258780411640139355
932648384020389066 760503454763580096
267219548490947717 93298917951520250
365858361851961921 597080998907420757
802247680183121157 881309814347453878
55179232080483060 76316874952809138
906925829283593803 537795789024700359
85560310302344049 784596403928377068
333578428120367881 730761127821537028
263519185214477509 843100645524737177
15290227822455100 29209685927003506
16761373508271054 839511068005865788
361767293221825598 655623123373814419
356690277741271392 285493300930960161
396757330844646394 616756900770235009
203952743504869889 451652743147541228
693688460136942979 248449066338884207
944971843399134283 161140358168624588
516471401363947470 352265507393429697
880856811334156591 32042299742856796
848986757414170648 682762575629612941
411184465369538675 188860718326437504
940842327059743063 486110753991253409
618747361136546437 452617687867264712
631751092829813372 293211458402168490
944380315235690040 903038484974353026
507962654014676724 983002727131870559
200021921627327742 873774946052717249
652528514471834141 263369479259129361
597054268922646094 115593048241570089
256123112677397171 891785554399455614
28319549084104798 909304604639853565
890876906730338327 426212002258268372
837882897082212763 623851113045146850
870166698779935429 685004080078694712
727425186631200821 553669963711742803
811686929960106752 367041516972317884
327896618276773784 349440509239550612
752436216061361510 147310634293933104
43132996963943428 912909971878285268
63645421067430773 804883117944235587
