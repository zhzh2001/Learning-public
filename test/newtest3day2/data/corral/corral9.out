744949921771806542
